src/AbstractResourceListener.php

```
@@ -144,6 +145,7 @@ public function getInputFilter()
      *
      * @param int $priority
      */
+    #[Override]
     public function attach(EventManagerInterface $events, $priority = 1)
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$priority'. (undefined)

(UnusedFormalParameter)